venues/EMNLP/python/init.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+
+import argparse
+import openreview
+import datetime
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print('connecting to {0}'.format(client.baseurl))
+
+emnlp_request_id = 'rylllwkbKN'
+
+author_consent_agreement = (
```

Comment:
We need to add a revision with these fields otherwise we will loose these changes the next time the builder create the conference without this script, for example: through the process function. 

@mohituniyal can help you to add 'Custom Submission Fields' to the revision invitation. 

One aside note: we should try to use the python convention to name the fields like 'custom_submissions_fields', then the UI will render them nicely, having spaces in json keys can be problematic. There is an open issue for Zach.